<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewObjectDialog</class>
 <widget class="QDialog" name="QNewObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLineEdit" name="typeEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>133</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="typeLabel">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="spriteBox">
         <property name="title">
          <string>Sprite</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="textureIDLabel">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="spriteCombo">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newSpriteButton">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="widthLabel">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="widthSpin">
              <property name="maximum">
               <number>99999</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="heightLabel">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="heightSpin">
              <property name="maximum">
               <number>99999</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boundingBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Bounding box</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="xBLabel">
                  <property name="text">
                   <string>x</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="xBSpin">
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="yBLabel">
                  <property name="text">
                   <string>y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="yBSpin">
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="wBLabel">
                  <property name="minimumSize">
                   <size>
                    <width>31</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Width</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="wBSpin">
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="hBLabel">
                  <property name="text">
                   <string>Height</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="hBSpin">
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="drawButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="S2PEditor.qrc">
                <normaloff>:/QObjectsDock/assets/images/GUI_Icons/edit.png</normaloff>:/QObjectsDock/assets/images/GUI_Icons/edit.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="valueGroupBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Special values</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QSpinBox" name="xSpeedSpin">
              <property name="minimum">
               <number>-99</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="ySpeedLabel">
              <property name="text">
               <string>Y speed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="xSpeedLabel">
              <property name="text">
               <string>X speed</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="value3Label">
              <property name="text">
               <string>Value3</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="value3Spin">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="ySpeedSpin">
              <property name="minimum">
               <number>-99</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="valueSpin">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="value2Spin">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="value2Label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Value2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="valueLabel">
              <property name="text">
               <string>Value</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="soundBox">
         <property name="title">
          <string>Sound effects</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="soundLabel">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Sound</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="soundCombo">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newSoundButton">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="sound2Label">
              <property name="minimumSize">
               <size>
                <width>44</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Sound2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sound2Combo">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="m_animBox">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>Animation frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="spriteSheetSArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>368</width>
            <height>281</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>typeEdit</tabstop>
  <tabstop>spriteCombo</tabstop>
  <tabstop>newSpriteButton</tabstop>
  <tabstop>widthSpin</tabstop>
  <tabstop>heightSpin</tabstop>
  <tabstop>xBSpin</tabstop>
  <tabstop>yBSpin</tabstop>
  <tabstop>wBSpin</tabstop>
  <tabstop>hBSpin</tabstop>
  <tabstop>drawButton</tabstop>
  <tabstop>xSpeedSpin</tabstop>
  <tabstop>ySpeedSpin</tabstop>
  <tabstop>value3Spin</tabstop>
  <tabstop>spriteSheetSArea</tabstop>
 </tabstops>
 <resources>
  <include location="S2PEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QNewObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QNewObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
